ui/components/app/snaps/snap-privacy-warning/snap-privacy-warning.js

```diff
@@ -33,119 +32,96 @@ export default function SnapPrivacyWarning({ onAccepted, onCanceled }) {
     useScrollRequired();
 
   return (
-    <Popover className="snap-privacy-warning">
-      <Box>
-        <Box
-          className="snap-privacy-warning__header"
-          paddingLeft={4}
-          paddingRight={4}
-        >
-          <Box
-            marginTop={4}
-            className="snap-privacy-warning__header__info-icon"
-            display={Display.Flex}
-            justifyContent={JustifyContent.center}
-            alignItems={AlignItems.center}
-          >
-            <AvatarIcon
-              iconName={IconName.Info}
-              color={IconColor.infoDefault}
-              backgroundColor={BackgroundColor.primaryMuted}
-              size={AvatarIconSize.Md}
-            />
-          </Box>
-          <Box
-            className="snap-privacy-warning__header__title"
-            marginTop={4}
-            marginBottom={4}
-            display={Display.Flex}
-            justifyContent={JustifyContent.center}
-            alignItems={AlignItems.center}
-          >
-            <Text variant={TextVariant.headingMd} fontWeight={FontWeight.Bold}>
-              {t('thirdPartySoftware')}
-            </Text>
-          </Box>
-        </Box>
-        <Box
-          paddingLeft={4}
-          paddingRight={4}
-          className="snap-privacy-warning__content"
-          ref={ref}
-          onScroll={onScroll}
-        >
-          <Box className="snap-privacy-warning__message">
-            <Text variant={TextVariant.bodyMd}>
-              {t('snapsPrivacyWarningFirstMessage', [
-                <ButtonLink
-                  key="privacyNoticeTermsOfUseLink"
-                  size={ButtonLinkSize.Inherit}
-                  href={TERMS_OF_USE_LINK}
-                  target="_blank"
+    <Modal onClose={onCanceled} isOpen className="snap-install-warning">
```

Comment:
I'm not sure if this was possible before, but you can click outside of this modal and the entire window closes.

Reply:
It was not possible before. I fixed it now. The thing is that the new Modal component requires `onClose` param.

Reply:
👍